react-common/components/share/ShareInfo.tsx

```diff
@@ -116,10 +117,33 @@ export const ShareInfo = (props: ShareInfoProps) => {
                 setKioskSubmitSuccessful(true);
                 try {
                     await addGameToKioskAsync(validKioskId, gameId);
+                    pushNotificationMessage({
+                        kind: 'info',
+                        text: lf("Game submitted to kiosk {0} successfully!", validKioskId),
+                        hc: false
+                    })
+
                 } catch (error) {
-                    pxt.log(error.message);
-                    //TODO: give some feedback to the user, no kiosk exists...
+                    if (error.message === "Not Found") {
```

Comment:
Have we considered telemetry for how often these types of errors occur? Maybe that would be too noisy, but something to consider...

The "Something went wrong" case might be of particular interest, though. Would be helpful if we could see what the request returned in the event, since it could indicate issues on our back end.

Reply:
Yeah, this is a good point. I would also want to avoid noisiness. I also realized, though, that I actually don't have any telemetry for when the submit is clicked, so I'll add that. At the very least, we want to know how many times a user clicked the kiosk button to get the input and actually submitted something. 

Similarly, I think it would be valuable to know how many of those submits successful and if there were server errors. Thanks for the input, Thomas!